pycqed/instrument_drivers/meta_instrument/qubit_objects/QuDev_transmon.py

```diff
@@ -2432,7 +2463,7 @@ def find_T1(self, times, n_cal_points_per_state=2, cal_states='auto',
                         prep_params=prep_params, label=label,
                         exp_metadata=exp_metadata)
 
-        #Extract T1 and T1_stddev from ma.T1_Analysis
+        # Extract T1 and T1_stddev from ma.T1_Analysis
         if analyze:
             T1_ana = tda.T1Analysis(qb_names=[self.name])
             if update:
```

Comment:
line 2476: should have one more tab, otherwise if analyze is false, T1 is undefined.